src/audio/Audio.js

```diff
@@ -78,13 +78,20 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 		}
 
+		if ( when === undefined || when === 0 ) when = this.context.currentTime;
+		this.startTime = offset !== undefined ? offset : this.startTime;
```

Comment:
How about...

```js
if ( offset === undefined ) offset = this.startTime;
```

I'm not sure we should mess with `this.startTime`, or if this is the right approach if we do.